app.py

```diff
@@ -63,7 +63,7 @@ def weather(city, state):
     #print out a statement with the current weather info + location that was used/detected
     #return("Right now in "+ city + ", " + state + " it is " + temperature +  degree_sign + " and there  is a " + RAIN_WARNING)
     location = {'city': city, 'state': state}
-    weather_info = {'temperature' : temperature, 'rain' : rain_commentary}
+    weather_info = {'temperature' : temperature, 'rain' : rain_commentary, 'celsius'=config.celsius}
```

Comment:
This throws a syntax error. There should be a `:` and not `=`. How were you about to get this running?

Reply:
oh correct. Will fix